protecode/scanning.py

```diff
@@ -121,6 +121,7 @@ def iter_components_with_vulnerabilities_and_assessments(
         self,
         products_to_import_from: list[pm.Product],
         use_product_cache: bool=True,
+        delete_inactive_products_after: int=0,
```

Comment:
codestyle: add unit as suffix (`_seconds`) . It would probably be nice to add a short docstr to the function where the parameters are briefly explained (specifically the cleanup-logic).

as for the default-value, rather use either `-1` or `None` (`0` might be a value a caller might want to set (generally speaking - _we_ do not plan to pass-in `0`, of course; if passing-in `0`, this would mean "delete immediately". which is kind of reasonable)